crates/swc_plugin_runner/src/lib.rs

```diff
@@ -153,18 +202,17 @@ struct PluginTransformTracker {
     instance: Instance,
     // Reference to the pointers succesfully allocated which'll be freed by Drop.
     allocated_ptr_vec: Vec<(i32, i32)>,
+    transform_result: Arc<Mutex<Vec<u8>>>,
```

Comment:
Is there a reason we can't use `Rc<RefCell<Vec<u8>>>`?
I checked the code and seems like it does not pass thread boundary (https://github.com/swc-project/swc/blob/1bb0dcb18f1ad33879f87304d187d5a449b6bc8c/crates/swc_plugin_runner/src/lib.rs#L328-L330)

Reply:
I also don't believe it doesn't pass thread boundary currently, but choose it for more of future precaution: wasm thread will be available, and we _may_ need to support it in somehow later - if that occurs we may need to revisit impl.

Reply:
so this was for guest (plugin) side thread boundary access instead of host side.

Reply:
I see, then I think it's fine.